### DISCUSSION TITLE
How do I enable the sync workflow?

### DISCUSSION THREAD

**Discussion by kunall-ll** [cecobask/imdb-trakt-sync#10] [General]
Enable the sync workflow, as scheduled workflows are disabled by default in fork repositories

How do i do this? I've set the workflow permissions to read and write permissions

**Comment by cecobask** [comment]
After creating a fork of the repository, do the following:
- Navigate to your fork
- Click on `Actions`
- Click on `I understand my workflows, go ahead and enable them` or whatever message is there to confirm
- Navigate to the `sync` workflow which can be found in your fork's `Actions` menu
- Click on `Enable workflow`

From this point the `sync` workflow will run automatically every 3 hours, but you can also trigger it manually by clicking on `Run workflow`.
Keep in mind you have to complete the section about repository secrets, outlined in the instructions for it to run successfully.

**Reply by kunall-ll** [reply]
Thanks. I've checked my secrets and I still get the following error:

{"level":"fatal","ts":"2023-01-[14](https://github.com/kunall-ll/imdb-trakt-sync/actions/runs/3916988946/jobs/6696497031#step:4:15)T05:01:18Z","caller":"syncer/syncer.go:63","msg":"failure initialising imdb client","error":"failure hydrating imdb client: failure scraping imdb watchlist id: imdb watchlist id not found: failure scraping imdb response for selector meta[property='pageId'] and attribute content","stacktrace":"github.com/cecobask/imdb-trakt-sync/pkg/syncer.NewSyncer\n\t/home/runner/work/imdb-trakt-sync/imdb-trakt-sync/pkg/syncer/syncer.go:63\nmain.main\n\t/home/runner/work/imdb-trakt-sync/imdb-trakt-sync/cmd/syncer/main.go:8\nruntime.main\n\t/opt/hostedtoolcache/go/1.18.9/x64/src/runtime/proc.go:250"}
exit status 1
Error: Process completed with exit code 1.
